parlai/core/torch_ranker_agent.py

```diff
@@ -267,7 +287,8 @@ def _build_candidates(self, batch, source, mode):
 
     @staticmethod
     def _find_match(cand_vecs, label_vec):
-        return ((cand_vecs == label_vec).sum(1) == cand_vecs.size(1)).nonzero()[0]
+        matches = ((cand_vecs == label_vec).sum(1) == cand_vecs.size(1)).nonzero()
+        return matches[0] if (len(matches.tolist()) > 0) else None
```

Comment:
do you really need the call to tolist()? len should work on the tensor directly and be faster, no?